server/vulnerabilities/oval/downloader.go

```diff
@@ -0,0 +1,62 @@
+package oval
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+const (
+	ovalSourcesFileName = "oval_sources.json"
+)
+
+// platform => web url
+type OvalSources map[Platform]string
+
+// The 'oval sources' is a metadata file hosted in the NVD repo, it contains
```

Comment:
The convention for go docs is:
```
// getOvalSources ...
func getOvalSources(...)
```
From https://go.dev/blog/godoc:
```
Notice this comment is a complete sentence that begins with the name of the element it describes. This important convention allows us to generate documentation in a variety of formats, from plain text to HTML to UNIX man pages, and makes it read better when tools truncate it for brevity, such as when they extract the first line or sentence.
```

Reply:
Cool, will update the comments, thanks